_docs/how-to/staking.md

```diff
@@ -0,0 +1,98 @@
+---
+title: FIO Staking Developer Guide
+description: FIO Staking Developer Guide
+redirect_from:
```

Comment:
You shouldn't need a redirect. These were put in place for people who had book marked pages on the previous Dev Hub site.